web/src/invite.ts

```diff
@@ -255,17 +255,20 @@ function generate_multiuse_invite(): void {
     });
 }
 
-function valid_to(time_valid: number): string {
-    if (!time_valid) {
-        return $t({defaultMessage: "Never expires"});
-    }
+function valid_to(time_valid: number, custom_expiration_time_input: number): string {
```

Comment:
The `custom_expiration_time_input` is a variable in this module, so you don't need to pass it to this function.

Could you refactor this function to take the value of the select element, which is a string, and then check for the two special values here: "null" and "custom"? And then you can use the `util` functions here to get the time in minutes directly in this function. This will consolidate the logic of getting this formatted string in this function.